### ISSUE TITLE
[UX] CKEditor block element formats drop-down: Change "Normal" to "Paragraph"

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop/backdrop-issues#2176]
CKEditor has a "Format" list allowing to select available HTML block elements. As noticed in https://github.com/backdrop/backdrop-issues/issues/2148#issuecomment-246611303, to select a `<p>` element, you have to choose "Normal". WordPress uses "Paragraph" instead of "Normal" which seems a better choice to me. Backdrop CMS should do the same.

@jenlampton assumed in https://github.com/backdrop/backdrop-issues/issues/2148#issuecomment-246869814 that the word 'Normal' doesn't come from Backdrop but from CKeditor itself, and asked to open a new issue for that. (That's that! I label the issue as feature request without milestone - feel free to change.)

**Comment by olafgrabienski** [comment]
Added "status - needs feedback". Dear reader, what do you think about the request?

**Comment by jenlampton** [comment]
I like this idea, but I wonder how hard it is to change the internals of ckeditor. Do we need to fix this upstream first?

**Comment by olafgrabienski** [comment]
Hm, what do you mean by "fix upstream"?

**Comment by jenlampton** [comment]
> Hm, what do you mean by "fix upstream"?

Open an issue against the ckeditor project and get them to fix it.

**Comment by olafgrabienski** [comment]
You think they're open to it?

**Comment by Dinornis** [comment]
I agree, the format should be labeled Paragraph rather than Normal.
Although this can be easily modified in any of the `/core/misc/ckeditor/lang/en-xx.js` files by changing the `"tag_p":"Normal"` string, it would be difficult to maintain in-between updates.

I searched the [CKEditor Issue Tracker](http://dev.ckeditor.com/), but could not find any requests for this change.  May be worth asking.

**Comment by olafgrabienski** [comment]
@Dinornis I searched the issue tracker too and found nothing related. However, at the moment, I've got simply no time to explain our request to another community. Would you mind doing so? Or anyone else?

**Comment by klonos** [comment]
https://dev.ckeditor.com/ticket/16824 😉

**Comment by Graham-72** [comment]
👍

**Comment by Dinornis** [comment]
Thanks for filing this @klonos 👍

**Comment by klonos** [comment]
No problem. Let's postpone then, pending on a reply from the CKEditor team. If they refuse to fix it upstream, then we can re-activate this issue and see if there's anything viable that we can do on our end.

**Comment by olafgrabienski** [comment]
Perfect, thanks!

**Comment by BWPanda** [comment]
Someone commented on @klonos' CKEditor ticket 6 years ago saying it was a good idea. But that's as far as it got. They've since moved their ticket queue to GitHub (https://github.com/ckeditor/ckeditor4/issues) and require users to re-submit feature requests there if still relevant. I couldn't find a similar, existing issue for this.

I do notice that the latest version of CKEditor v4 (as per their demo) still uses the term 'Normal': https://ckeditor.com/ckeditor-4/demo/
Whereas their CKEditor v5 has renamed it to 'Paragraph': https://ckeditor.com/ckeditor-5/demo/feature-rich/

So we can either:
1. Re-submit the request in the CKEditor v4 issue queue and hope they're still adding new features to that version, or
2. Just wait until [Backdrop supports CKEditor v5](https://github.com/backdrop/backdrop-issues/issues/4122) and inherit this fix as a bonus.

**Comment by olafgrabienski** [comment]
Thanks for checking! I'd say, let's wait then!

**Comment by BWPanda** [comment]
In that case I think we can close this issue then, in favour of https://github.com/backdrop/backdrop-issues/issues/4122

**Comment by klonos** [comment]
FTR, since switching to CKEditor 5 may take a while, I have raised https://github.com/ckeditor/ckeditor4/issues/5428 upstream 😉
